src/views/Farms/Farms.tsx

```diff
@@ -130,13 +130,23 @@ const Farms: React.FC = () => {
   }, [account, dispatch, fastRefresh])
 
   const [stackedOnly, setStackedOnly] = useState(false)
+  const isActive = !pathname.includes('history')
 
-  const activeFarms = farmsLP.filter((farm) => farm.pid !== 0 && farm.multiplier !== '0X')
-  const inactiveFarms = farmsLP.filter((farm) => farm.pid !== 0 && farm.multiplier === '0X')
+  const filterFarms = (farms: FarmWithStakedValue[]): FarmWithStakedValue[] => {
+    let result
```

Comment:
Avoid the use of let, and variable re assignation, it's one of the basic good practice of JS development